.github/scripts/check_labels.py

```diff
@@ -10,20 +11,46 @@
     GitRepo,
 )
 from trymerge import (
+    _fetch_url,
     gh_post_pr_comment,
     GitHubPR,
 )
 
 
+ERR_MSG = ("This PR needs a label. If your changes are user facing and intended to be a "
+           "part of release notes, please use a label starting with `release notes:`. If "
+           "not, please add the `topic: not user facing` label. For more information, see "
+           "https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work.")  # noqa: E501  pylint: disable=line-too-long
+
+
 def get_release_notes_labels() -> List[str]:
     return [label for label in get_pytorch_labels() if label.lstrip().startswith("release notes:")]
 
 
-def check_labels(pr_labels: List[str]) -> bool:
+def delete_comment(comment_id: int) -> None:
+    url = f"https://api.github.com/repos/pytorch/pytorch/issues/comments/{comment_id}"
+    _fetch_url(url, method="DELETE")
+
+
+def check_labels(pr: GitHubPR) -> bool:
+    pr_labels = pr.get_labels()
+
+    # Check if PR is not user facing
     is_not_user_facing_pr = any(label.strip() == "topic: not user facing" for label in pr_labels)
     if is_not_user_facing_pr:
         return True
 
+    # Check if bot has already posted message within the past hour to include release notes label
+    err_msg_start = ERR_MSG.partition(".")[0]
```

Comment:
Perhaps it's better to pull the message title "This PR needs a label" into a separate const variable and reference that in both ERR_MSG and here